### DISCUSSION TITLE
Working serial numbers

### DISCUSSION THREAD

**Discussion by Scrath1** [dewenni/ESP32-Jarolift-Controller#12] [Q&A] [answered]
Hello,
I am currently trying to set this project up to test it before migrating from my existing Jarolift-MQTT transmitter.
Because of that I am trying to use a new serial number with the set up in order to not mess up my old transmitters device counter.

Right now I can't get the new transmitter paired to my shutters. I remember that they were kind of picky regarding the serial number prefix used and am currently trying to find ones which have been confirmed to work by others.

My previous controller used the serial number `0x12deaf` successfully.

If you could share serial numbers that have worked for you I'd be grateful.

**Comment by flushbug** [comment, marked as answer]
000010 is working fine in my case

**Comment by dewenni** [comment]
I was also able to use my previous serial number in combination with the existing device counter value and no need to learn the shutters again.

see also here: https://github.com/dewenni/ESP32-Jarolift-Controller?tab=readme-ov-file#migration

**Reply by wunderbaum** [reply]
0x010016 is in my old one.
